bigquery_etl/view/generate_stable_views.py

```diff
@@ -345,15 +345,14 @@ def main():
         last for k, (*_, last) in groupby(schemas, lambda t: t.bq_dataset_family)
     ]
 
-    with ThreadPool(args.parallelism) as pool:
+    with Pool(args.parallelism) as pool:
         pool.map(
             partial(
                 write_view_if_not_exists,
                 args.target_project,
                 Path(args.sql_dir),
             ),
             schemas,
-            chunksize=1,
```

Comment:
Was there a reason for setting `chunksize=1` here? It does slow the whole process down by a lot.